openapi3filter/validation_kit.go

```diff
@@ -0,0 +1,85 @@
+package openapi3filter
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+)
+
+///////////////////////////////////////////////////////////////////////////////////
+// We didn't want to tie kin-openapi too tightly with go-kit.
```

Comment:
Let's not copy https://github.com/go-kit/kit/blob/266ff8dc37c693d0649707e519c93c1f85868bdc/transport/http/server.go here but rather import this package.

Reply:
On second thought this brings a pretty large dependency tree so disregard the above comment.